app/globals.d.ts

```diff
@@ -3,5 +3,10 @@ declare module '*.css' {
   export default classes;
 }
 
+declare module '*.png' {
```

Comment:
Could you not add the other image file formats that we use as well?

Reply:
Done :handshake: